apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -863,14 +863,14 @@ const Main = ({
   const isMobile = useMediaQuery("(max-width: 768px)");
   const searchParams = useCompatSearchParams();
 
-  if (!rawData || status === "pending") {
```

Comment:
!rawData is true when an error happens, which meant the error was never displayed and the SkeletonLoader was always show.